<ui version="4.0" >
 <class>calculateForm</class>
 <widget class="QWidget" name="calculateForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Risk Initial:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="riskLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Risk Reduced:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="riskReducedLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Likelihood Initial:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="vulnerabilityLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Likelihood Reduced:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="vulnerabilityReducedLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Max Budget: $</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="maxBudgetLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Budget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="budgetSpinBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix" >
        <string/>
       </property>
       <property name="prefix" >
        <string>($) </string>
       </property>
       <property name="maximum" >
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QSlider" name="budgetSlider" >
     <property name="maximum" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QToolButton" name="calculateButton" />
     </item>
     <item>
      <widget class="QLabel" name="infoLabel" >
       <property name="text" >
        <string>Calculate Allocation for
Current Fault Tree</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>240</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>budgetSlider</tabstop>
  <tabstop>calculateButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>budgetSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>budgetSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>95</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel" >
     <x>92</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>budgetSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>budgetSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>183</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel" >
     <x>186</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
